apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -13,6 +13,35 @@ import HeaderBannerNoImage from '@cdo/apps/templates/HeaderBannerNoImage';
 import TwoColumnActionBlock from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
 import ActionBlocksWrapper from '@cdo/apps/templates/studioHomepages/ActionBlocksWrapper';
 import style from './landingPage.module.scss';
+import Tabs from '@cdo/apps/componentLibrary/tabs';
+
+const DEFAULT_SELECTED_TAB = 'myPL';
+
+const getAvailableTabs = () => {
+  // [TODO]: return a subset of the tabs below based on the user's permission level
```

Comment:
I love this approach, but I don't think we want to launch with showing them all. I know this was a bit of a weird task in that sense, but I think, for now, we should only show the `myPL` tab and we can add the rest as we implement them?

I think all I'm asking for here is to remove/comment out all but the first option. The strings and everything can stay!